### PR TITLE
[NNCF] Filter pruning

### PR DESCRIPTION
**Pull request by alexsu52**

### CODE REVIEW

**Comment by nervana-ff** [comment]
Can one of the admins verify this patch?

**Comment by sovrasov** [comment]
Task and configs were removed from this repo, please move the changes to ote: https://github.com/openvinotoolkit/training_extensions/pull/972

File: torchreid/integration/nncf/accuracy_aware_training/__init__.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Copyright (C) 2021 Intel Corporation
```
Comment (sovrasov) [inline torchreid/integration/nncf/accuracy_aware_training/__init__.py]: This license header looks outdated. We use
```
# Copyright (C) 2020-2022 Intel Corporation
# SPDX-License-Identifier: Apache-2.0
```

File: torchreid/integration/nncf/accuracy_aware_training/common/runner.py [hunk 2]
```diff
@@ -0,0 +1,327 @@
...
+from nncf.common.utils.backend import infer_backend_from_compression_controller
+from nncf.common.utils.backend import BackendType
+
+ModelType = TypeVar('ModelType')
+OptimizerType = TypeVar('OptimizerType')
+LRSchedulerType = TypeVar('LRSchedulerType')
+TensorboardWriterType = TypeVar('TensorboardWriterType')
+
+
+class TrainingRunner(ABC):
```
Comment (sovrasov) [inline torchreid/integration/nncf/accuracy_aware_training/common/runner.py]: Are this train runner abstract class and other abstract classes unique for d-o-r? Seems like this stuff could be located somewhere inside nncf package.

File: torchreid/integration/nncf/accuracy_aware_training/__init__.py (hunk 1 above)
Comment (alexsu52) [reply torchreid/integration/nncf/accuracy_aware_training/__init__.py]: done

File: torchreid/integration/nncf/accuracy_aware_training/common/runner.py (hunk 2 above)
Comment (alexsu52) [reply torchreid/integration/nncf/accuracy_aware_training/common/runner.py]: done

**Review by sovrasov — APPROVED** [review]

**Pull request merged**